### ISSUE TITLE
Prevent hard-crash on OpenJDK environments on launch

### ISSUE THREAD

**Issue by Col-E** [Col-E/Recaf#79]
**Describe the bug**

> Hard crash when the OpenJDK project is installed but OpenJFX is not

```
[~/Downloads]
(zsh) $ java -jar recaf-1.5.2.jar 
Exception in thread "main" java.lang.NoClassDefFoundError: javafx/application/Application$Parameters
    at java.lang.Class.getDeclaredMethods0(Native Method)
    at java.lang.Class.privateGetDeclaredMethods(Class.java:2701)
    at java.lang.Class.getDeclaredMethods(Class.java:1975)
    at me.coley.event.Bus.subscribe_(Bus.java:56)
    at me.coley.event.Bus.subscribe(Bus.java:28)
    at me.coley.recaf.Recaf.main(Recaf.java:32)
Caused by: java.lang.ClassNotFoundException: javafx.application.Application$Parameters
    at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
    at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
    ... 6 more
[~/Downloads]
(zsh) $ java -version
openjdk version "1.8.0_181"
OpenJDK Runtime Environment (build 1.8.0_181-8u181-b13-1~deb9u1-b13)
OpenJDK 64-Bit Server VM (build 25.181-b13, mixed mode)
```

**To Reproduce**
1. Run recaf on `openjdk` without `openjfx` being installed

****

Rather than a crash, an alert should be shown to notify users.

**Comment by Col-E** [comment]
Resolved with 361e065fa9a3efc1aeba0a08a0da3155ced31b45

Checks for environments known to cause issues.

**Comment by FFY00** [comment]
Just a small tip. If you add something like `closes #79` or `resolves #79` to the commit body, either title or description, github will automatically close the issue for you. When closing an issue thi was, github will add a link to the commit in question.

You can read a more detailed explanation here.
https://help.github.com/articles/closing-issues-using-keywords/

Hope it helps :smile:.
